content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -122,7 +122,7 @@ second line.}
 
 ### Contextual Logging
```

Comment:
(nit / good to fix before GA / yes, I know the rest of the file is also not following the style guide here)
```suggestion
### Contextual logging
```